CONTRIBUTING.md

```diff
@@ -102,6 +102,14 @@ We generally allow backports of following changes only to the latest supported b
 - Improvements to CI signal
 - Improvements to the test framework
 
+While we recommend to target following type of changes to the next minor release, CAPI maintainers will also consider allowing
+backport of following changes only to the latest supported branch:
+- Small enhancements or additions to experimental Cluster API features, with the goal of allowing faster adoption and iteration;
```

Comment:
"Small" is subjective, what is considered a "small" change?

Reply:
Dropped small; I think that what is written after about the focus on stability on the release branch and the impact of those backports being "limited and well-scoped" already conveys the key points